src/components/ChallengeEditor/ChallengeViewTabs/index.js

```diff
@@ -85,17 +85,17 @@ const ChallengeViewTabs = ({
       <div className={styles.topContainer}>
         <div className={styles.leftContainer}>
           <div className={styles.title}>{challenge.name}</div>
-          {!isTask && (
```

Comment:
@dedywahyudi I don't think we need to show legacy links to all types. This change would make legacy links visible to every challenge type.